deployments/helm/templates/p2p-service.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and (.Values.backup.enabled) .Values.p2p.enabled }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "papyrus.name" . }}-p2p-svc
```

Comment:
no need to add `svc` suffix (no need to mention the resource type in the resource name)